db/migrate/20170216193758_add_vm_table_comments.rb

```diff
@@ -0,0 +1,81 @@
+class AddVmTableComments < ActiveRecord::Migration[5.0]
+  def change
+    change_table_comment :vms, "Virtual Machines"
+
+    change_column_comment :vms, :autostart, "A boolean value that indicates whether or not the VM is set to autostart."
+    change_column_comment :vms, :availability_zone_id, "FK to the AVAILABILITY_ZONES table."
+    change_column_comment :vms, :busy, "HELP"
+    change_column_comment :vms, :boot_time, "The last time the VM was started."
+    change_column_comment :vms, :cloud, "HELP"
+    change_column_comment :vms, :cloud_network_id, "FK to the CLOUD_NETWORKS table."
+    change_column_comment :vms, :cloud_subnet_id, "FK to the CLOUD_SUBNETS table."
+    change_column_comment :vms, :cloud_tenant_id, "FK to the CLOUD_TENANTS table."
+    change_column_comment :vms, :config_xml, "HELP"
+    change_column_comment :vms, :connection_state, "HELP"
+    change_column_comment :vms, :cpu_affinity, "HELP"
+    change_column_comment :vms, :cpu_hot_add_enabled, "HELP"
+    change_column_comment :vms, :cpu_hot_remove_enabled, "HELP"
+    change_column_comment :vms, :cpu_limit, "HELP"
+    change_column_comment :vms, :cpu_reserve, "HELP"
+    change_column_comment :vms, :cpu_reserve_expand, "HELP"
+    change_column_comment :vms, :cpu_shares, "HELP"
+    change_column_comment :vms, :cpu_shares_level, "HELP"
+    change_column_comment :vms, :created_on, "The date the VM was added to ManageIQ inventory."
+    change_column_comment :vms, :deprecated, "HELP"
+    change_column_comment :vms, :description, "A custom description that users can add within ManageIQ."
+    change_column_comment :vms, :ems_created_on, "Date the associated EMS was created."
+    change_column_comment :vms, :ems_id, "FK to the EXT_MANAGEMENT_SYSTEMS table."
+    change_column_comment :vms, :evm_owner_id, "HELP"
+    change_column_comment :vms, :ems_ref_obj, "HELP"
+    change_column_comment :vms, :ems_cluster_id, "HELP"
+    change_column_comment :vms, :fault_tolerance, "HELP"
+    change_column_comment :vms, :flavor_id, "FK to the FLAVORS table."
+    change_column_comment :vms, :format, "HELP"
+    change_column_comment :vms, :guid, "An identifier used to uniquely identify the VM internally."
+    change_column_comment :vms, :id, "The internal database ID."
+    change_column_comment :vms, :host_id, "FK to the HOSTS table."
```

Comment:
I feel like these foreign keys need more description than just "an FK".  For example, this could be `The host that this VM/Template currently resides on.  FK to the hosts table`